### ISSUE TITLE
Transport Cookies not passed -> Could not assign Workitem

### ISSUE THREAD

**Issue by schallinterferenz** [jesper-raemaekers/python-polarion#61]
Hello,

We have a 2 server setup for Polarion in AWS, and the 2 servers run behind a load balancer.
The problem we encounter is that it happes often that we authenticate on one of the servers, and the next request goes to the other server and we get the following error:

```
Tag not found in polarion: Cannot find workitem <workitem_id> in project <project>
[2022-04-05 09:39:32,016] ERROR in app: Exception on /post [POST]
Traceback (most recent call last):
File "/home/eduard/.local/lib/python3.9/site-packages/flask/app.py", line 2446, in wsgi_app
response = self.full_dispatch_request()
File "/home/eduard/.local/lib/python3.9/site-packages/flask/app.py", line 1951, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/home/eduard/.local/lib/python3.9/site-packages/flask/app.py", line 1820, in handle_user_exception
reraise(exc_type, exc_value, tb)
File "/home/eduard/.local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
raise value
File "/home/eduard/.local/lib/python3.9/site-packages/flask/app.py", line 1949, in full_dispatch_request
rv = self.dispatch_request()
File "/home/eduard/.local/lib/python3.9/site-packages/flask/app.py", line 1935, in dispatch_request
return self.view_functions[rule.endpoint](**req.view_args)
File "/home/eduard/Repos/efg_devops_git-polarion-bridge/./github_polarion_bridge.py", line 433, in hook_github
add_polarion_table_entry(polarion_client, work_item, commit,
UnboundLocalError: local variable 'work_item' referenced before assignment
127.0.0.1 - - [05/Apr/2022 09:39:32] "POST /post HTTP/1.1" 500 -
```

My question now is, how I can pass the transport cookiejar, as well as the session id, via the library, so that our load balacer knows where to route the request?

**Comment by schallinterferenz** [comment]
Hello jesper, 

Thank you for the quick response. Do I have to specify the cookies somewhere, after you updated the lib?

**Comment by jesper-raemaekers** [comment]
Hi,

I expected the cookies to be assigned to the first call (logging in) and took them from there. (see https://github.com/jesper-raemaekers/python-polarion/commit/dffa8ad126a4e3054b6694aff68007b1326cb98b#diff-9e4a1f402680e813c78e2a66d9b8b7ff039fa510dac419d880a8968327ce3ab8R92 )
But it seems i might misunderstand, you want to specify a cookie? How do you create this?

**Comment by schallinterferenz** [comment]
I think I downloaded the false version :D I pip installed from main instead of the new branch. 
I will now try to install the version with your changes and will answer you as soon as possible. 
Thx a lot for the fast response!

**Comment by schallinterferenz** [comment]
Works like a charm :) Tested it five time. No errors at all. It finds the project instantly and our load balancer gets the right cookies. 

Thanks a lot!

**Comment by schallinterferenz** [comment]
When do you think you will release a update?

**Comment by jesper-raemaekers** [comment]
It was released today with version 0.4.0

**Comment by schallinterferenz** [comment]
Thanks a lot!
